my-app/src/App/pages/CoinPage/CoinPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+
+import { Stocks } from "@config/const";
+import { useParams } from "react-router-dom";
+
+import styles from "./CoinPage.module.scss";
+import Coin from "./components/Coin/Coin";
+import { useCoinContext } from "../../App";
+
+const CoinPage = () => {
+  const [coin, setCoin] = useState<Stocks>({
+    id: "",
+    name: "",
+    image: "",
+    current_price: 0,
+    symbol: "",
+    price_change_percentage_24h: 0,
+    price_change_24h: 0,
+    market_cap: 0,
+    fully_diluted_valuation: 0,
+    circulating_supply: 0,
+    total_supply: 0,
+    max_supply: 0,
+  });
+  const coinContext = useCoinContext();
+
+  const { coinID } = useParams();
+
+  let stock: any = coinContext.find((item) => item.id === coinID);
```

Comment:
давай без `any` пожалуйста( и `let` тут ни к чему, надо `const` использовать. и хорошо бы обернуть в `React.useMemo` https://beta.reactjs.org/reference/react/useMemo
```suggestion
  const stock = React.useMemo(() => coinContext.find((item) => item.id === coinID), [coinContext, coinID]);
```